plugins/nodes/src/js/data/NetworkNodesClient.ts

```diff
@@ -0,0 +1,35 @@
+import { Observable, merge } from "rxjs";
+import { request, RequestResponse } from "@dcos/http-service";
+import { partition, tap } from "rxjs/operators";
+
+export interface NetworkNode {
+  updated: string;
+  public_ips: string[];
+  private_ip: string;
+  hostname: string;
+}
+
+export type NetworkNodesResponse = NetworkNode[];
+
+export function fetchNetworkNodes(): Observable<RequestResponse<string[]>> {
+  const [success, error] = partition(
+    (response: RequestResponse<any>) => response.code < 300
```

Comment:
shouldn't it be `RequestResponse<NetworkNodesResponse>`?

Reply:
addressed